botorch/acquisition/acquisition.py

```diff
@@ -101,7 +101,11 @@ def forward(self, X: Tensor) -> Tensor:
 
 
 class OneShotAcquisitionFunction(AcquisitionFunction, ABC):
-    r"""Abstract base class for acquisition functions using one-shot optimization"""
+    r"""
+    Abstract base class for acquisition functions using one-shot optimization
```

Comment:
```suggestion
    r"""Abstract base class for acquisition functions using one-shot optimization.
```
Is there a particular reason for having these in a new line? I think we generally use the line starting with `r"""` for the one-line description.

Reply:
Sorry, I somehow missed these comments... this one isn't showing up in the "Files changed" pane, and it looks like I made the opposite change?

In any case, PEP 257 says "The summary line may be on the same line as the opening quotes or on the next line." pydocstyle additionally specifies "One-line docstring should fit on one line with quotes." but doesn't seem to care if a multi-line docstring starts with a newline. This is rendering fine.